src/runner/driver-test-runner.js

```diff
@@ -90,30 +90,40 @@ export class DriverTestRunner {
   /**
    * Used for v2 tests to ensure proper settings.
    *
-   * @param {string} settings - "browseMode" "focusMode" for NVDA, "pcCursor" "virtualCursor"
-   *                 for JAWS., "defaultMode" for others.
+   * @param {string} settings - space seperated list of settings. "browseMode", "focusMode" for NVDA,
+                                "quickNavOn", "defaultMode", etc for VoiceOver.
    */
   async ensureSettings(settings) {
     const { atName } = await this.collectedCapabilities;
-    if (atName == 'NVDA') {
-      const desiredResponse = { browsemode: 'Browse mode', focusmode: 'Focus mode' }[
-        settings.toLowerCase()
-      ];
-      if (!desiredResponse) {
-        throw new Error(`Unknown command settings for NVDA "${settings}"`);
-      }
+    // break up the space-separated settings into an array so we can iterate over it
+    const settingsArray = settings.split(' ');
 
+    if (atName == 'NVDA') {
       // disable the "beeps" when switching focus/browse mode, forces it to speak the mode after switching
       await this.atDriver._send({
         method: 'nvda:settings.setSettings',
         params: { settings: [{ name: 'virtualBuffers.passThroughAudioIndication', value: false }] },
       });
 
       try {
-        await this.pressKeysToToggleSetting(
-          ATKey.sequence(ATKey.chord(ATKey.key('insert'), ATKey.key('space'))),
-          desiredResponse
-        );
+        for (const setting of settingsArray) {
+          switch (setting.toLowerCase()) {
```

Comment:
for the purposes of this control flow, we could ditch the lower casing and update the case statements but i was wary of doing so in case configuration elsewhere assumes that all lowercase is valid.